namespace cpp FrontEndOps

service FrontEndOps {
    void setAlive(1: bool isAlive)
}
